service/controller/key/common.go

```diff
@@ -72,6 +72,11 @@ func AWSTags(getter LabelsGetter, installationName string) map[string]string {
 	return tags
 }
 
+func AvailabilityZoneRegionSuffix(az string) string {
+	elements := strings.Split(az, "-")
+	return elements[len(elements)-1]
```

Comment:
this could use little bit of comment with an example, I had to read it several times and know what values is going in to figure out what it will return